kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -480,7 +480,7 @@ private static boolean loadFromKubeconfig(Config config, String context, String
           config.setUsername(currentAuthInfo.getUsername());
           config.setPassword(currentAuthInfo.getPassword());
 
-          if (Utils.isNullOrEmpty(config.getOauthToken()) && currentAuthInfo.getAuthProvider() != null && !Utils.isNullOrEmpty(currentAuthInfo.getAuthProvider().getConfig().get(ACCESS_TOKEN))) {
+          if (Utils.StringUtils.isNullOrEmpty(config.getOauthToken()) && currentAuthInfo.getAuthProvider() != null && !Utils.StringUtils.isNullOrEmpty(currentAuthInfo.getAuthProvider().getConfig().get(ACCESS_TOKEN))) {
```

Comment:
I am not a big fan of this nesting.

We should either leave string utilities under utils, it have StringUtils a separate file.